docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -98,6 +98,12 @@ If thresholds are set, then the field background is displayed in the appropriate
 
 {{< figure src="/static/img/docs/tables/color-background.png" max-width="500px" caption="Color background" class="docs-image--no-shadow" >}}
 
+By selecting the **Apply to entire row** option, the entire row will be colored with the background color configured for this cell.
+
+{{< figure src="/static/img/docs/tables/apply-to-row.png" max-width="500px" caption="Apply to entire row" class="docs-image--no-shadow" >}}
```

Comment:
I don't think we need the image of the setting since we don't generally do that. If you want to show an image, I'd say let's do that in the What's new entry.

```suggestion
```

Reply:
Makes sense to me!